projects/laji/src/i18n/fi.json

```diff
@@ -2079,4 +2082,4 @@
   "whatsNew.message": "laji.fi on Suomen Lajitietokeskuksen testikäytössä oleva portaali. Tietoja täydennetään ja ominaisuuksia parannetaan jatkuvasti. Palaute on tervetullutta - sitä voi lähettää palautelomakkeella tai osoitteeseen info@laji.fi. Kiitos!<p>Voit katsoa tuoreimmat muutokset <a href=\"https://laji.fi\">etusivun</a> ajankohtaista osiosta.</p",
   "wholeYear": "Koko vuosi {{year}}",
   "yes": "Kyllä"
-}
```

Comment:
Do we have guidelines on whether to use newline at the end of files or not?

Reply:
We don't have any general guidelines but I think most of the time we have newline at the end